.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -105,6 +109,49 @@ steps:
           imagePrefix: "${IMAGE_MACOS_X86_64}"
         artifact_paths: "auditbeat/build/*.xml"
 
+      - label: ":linux: {{matrix.image}} Integration Tests"
+        key: "linux-integration"
+        command: ".buildkite/auditbeat/scripts/integ-tests.sh"
+        notify:
+          - github_commit_status:
+              context: "auditbeat/Extended: Linux Integration Tests"
```

Comment:
nit: use `auditbeat: Integration Tests`, for consistency?

Reply:
I've changed to this name